host.json

```diff
@@ -11,6 +11,19 @@
   "extensionBundle": {
     "id": "Microsoft.Azure.Functions.ExtensionBundle",
     "version": "[4.0.0, 5.0.0)"
+  },
+  "extensions": {
+    "queues": {
+      "maxPollingInterval": "00:00:15",
+      "visibilityTimeout" : "00:00:30",
+      "batchSize": 16,
+      "maxDequeueCount": 5,
```

Comment:
Are we ok with retrying a job 5 times? If so we need to address a situation where a job fails from a bug since retrying will always result in a failure

Reply:
I'll cap it at 2.
errors could arise from calls too openai.